tpcc/check.go

```diff
@@ -93,16 +91,21 @@ func checkConsistency(db *sql.DB) error {
 
 	// 3.3.2.4
 	// sum(O_OL_CNT) = # of rows in the ORDER-LINE table for each warehouse/district
-	var sumOOLCnt int
+	var sumOOLCnt, countNewOrderLint int
```

Comment:
why is this called Lint? just curious

Reply:
Because I'm having trouble spelling "line" today 😄

Reply:
Oh, I get it, haha